source/_integrations/media_source.markdown

```diff
@@ -24,28 +24,28 @@ media_source:
 
 ## Local Media
 
-By default, a local media source is configured which Home Assistant looks for
+By default, the integration creates a local folder where Home Assistant looks for
```

Comment:
This is incorrect. It is configured by default. The core does not create the folder.